contracts/ColonyNetworkENS.sol

```diff
@@ -88,6 +88,30 @@ contract ColonyNetworkENS is ColonyNetworkStorage {
     emit ColonyLabelRegistered(msg.sender, subnode);
   }
 
+  function updateColonyOrbitDB(string memory orbitdb)
```

Comment:
Do we care about protecting against an empty `orbitdb` string?

Reply:
I don't think so, though I don't feel strongly about it. An empty string vs a string that isn't an `orbitdb` address seems like a negligible difference for the dApp to deal with.